service/src/main/kotlin/app/cash/backfila/dashboard/StopAllBackfillsAction.kt

```diff
@@ -47,7 +47,7 @@ class StopAllBackfillsAction @Inject constructor(
     }
 
     runningBackfillIds.forEach { backfillId ->
-      logger.info { "Stop backfill ${backfillId} by ${caller.get()?.user}" }
+      logger.info { "Stop backfill $backfillId by ${caller.get()?.user}" }
       backfillStateToggler.toggleRunningState(backfillId, caller.get()!!, BackfillState.PAUSED)
     }
```

Comment:
I think I also want to make sure that slack messaging is tested when we have an action like this. Also I think we might want to change the slack message a bit. But that isn't part of this PR.